cinema_service/urls.py

```diff
@@ -3,6 +3,7 @@
 
 urlpatterns = [
     path("admin/", admin.site.urls),
+    path("api/user/", include("user.urls", namespace="user")),
```

Comment:
The `include` function for `user.urls` uses the `namespace` argument. Ensure that the `user.urls` module has an `app_name` defined, as this is required when using namespaces in Django URL configurations. This is correctly set in the `user/urls.py` file.